utils/clflags.ml

```diff
@@ -222,7 +222,7 @@ let afl_inst_ratio = ref 100           (* -afl-inst-ratio *)
 
 let function_sections = ref false      (* -function-sections *)
 let probes = ref Config.probes         (* -probes *)
-let infer_with_bounds = ref false       (* -infer-with-bounds *)
+let infer_with_bounds = ref true       (* -infer-with-bounds *)
```

Comment:
if we plan on releasing this, I think we should do the proper thing and make the flag an opt-out. Or just remove it entirely.